docs/Credits.md

```diff
@@ -29,11 +29,56 @@ Soundtrack and Sound Effects
 * Camera Click Noise from Snapper4298, from FreeSound.org
 * Other sound effects created by Exile.
 
+Icons
+--------
+
+Some NUI editor icons belong to ["Fugue Icons" by Yusuke Kamiyamane](http://p.yusukekamiyamane.com/), licensed under the [Creative Commons Attribution 3.0 License](https://creativecommons.org/licenses/by/3.0/):
+* editor_array.png
+* editor_attribute.png
+* editor_object.png
+* editor_uibox.png
+* editor_uibutton.png
+* editor_uicheckbox.png
+* editor_uidoubleslider.png
+* editor_uidropdown.png
+* editor_uidropdownscrollable.png
+* editor_uiimage.png
+* editor_uilabel.png
+* editor_uilist.png
+* editor_uiloadbar.png
+* editor_uiscrollbar.png
+* editor_uislider.png
+* editor_uitext.png
+* editor_uitextentry.png
+* editor_uitooltip.png
+* editor_uitreeview.png
+* editor_zoomablelayout.png
+
+
+Some icons have been made by Florian K�berle and are licensed under the [Creative Commons Attribution 3.0 License](https://creativecommons.org/licenses/by/3.0/):
```

Comment:
The second character in your surname (assuming that's ö) is replaced by � (desktop Chrome). Might be an encoding issue?